cmd/protoc-gen-openapi/examples/tests/protobuftypes/openapi.yaml

```diff
@@ -246,7 +246,7 @@ components:
                 - type: array
                   items:
                     $ref: '#/components/schemas/AnyJSONValue'
-            description: AnyJSONValue is a "catch all" type that can hold any JSON value, except null as this is not allowed in OpenAPI
+            description: '`Value` represents a dynamically typed value which can be either null, a number, a string, a boolean, a recursive struct value, or a list of values. A producer of value is expected to set one of that variants, absence of any variant indicates an error. The JSON representation for `Value` is JSON value.'
```

Comment:
This description is coming straight off of the google.protobuf.Value message here:
https://github.com/protocolbuffers/protobuf/blob/main/src/google/protobuf/struct.proto#L56L61
Which followed the pattern of using the message comments for the schema descriptions.

I can switch it back to a hard coded value though if that is preferred.

Reply:
When running the schema through a JSON validator, you will get an error, if the value is `null`, but not if the property is not set at all.
So basically: it can not be null - it is our definition that is wrong, since it does not have `null` as a possible type, but we can not have `null` as a type, because OpenAPI v3.0.x does not allow that.
It will be doable in OpenAPI v3.1.0 as full JSON Schema is supported.

@jeffsawatzky and @timburks what is your opinions here?

Reply:
I'm not actually a big fan of the comment for the `Any` message...it's way too long. So I might hard code that to something smaller anyway. If you want me to change the comment here for `Value` too I can do that.

Reply:
As it is actually incorrect, due to the need of breaking that definition, I think it would make sense to use something like the original `AnyJSONValue is a "catch all" type that can hold any JSON value, except null as this is not allowed in OpenAPI`.
Maybe borrow some from the protobuf comment:
`Value represents a dynamically typed value which can be any JSON value, except null`.

Reply:
For both `AnyJSONValue` and `Value`

Reply:
@morphar I was referring to this big long comment for the `Any` message. It's way too huge.
https://github.com/google/gnostic/pull/327/files#diff-190e6a851c89c364cd83e2bd05fc44555cdb0d940341e5c531a4eae9c0c66308R29

Reply:
Aaahh... Gotcha :) 👍

Reply:
Also, while the "null" type is not allowed, we could make the schema [nullable](https://swagger.io/specification/#fixed-fields-20) which would allow null values.

Reply:
Nice catch!
That would also allow for proper handling of `google.protobuf.Value`, as it can then also be set to `required`:
`... absence of any variant indicates an error ...` 👍